.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,20 +16,22 @@ This section describes how to determine whether a blueprint has enough value to
 This section describes how to determine whether the blueprint has the right technical quality to be included in the central XebiaLabs blueprints repository.
 
 - [ ] The branch from which the PR is created is up-to-date with the `development` branch.
+- [ ] There is a `README.md` file at the root of the blueprint folder that describes the blueprint, using the standard README template.
 - [ ] The blueprint generates a `xebialabs/README.md` file which, at the minimum, explains how to use the blueprint after it is generated (like adding any missing steps, creating accounts, setting up docker containers, applying the YAML using `xl` CLI, running release etc). **N.B.:** Do not use this document to describe how to instantiate the blueprint. It will only be avialable to the user *after* the blueprint has been instantiated.
 - [ ] The blueprint generates a `docker/docker-compose.yml` file which can be used to spin up the required non-XebiaLabs products that the blueprints uses. This can be used to test and/or try out the blueprint without having to manually install those products.
-- [ ] The blueprint does not contain sensitive information such passwords, tokens, credentials or licences. 
+- [ ] The blueprint does not contain sensitive information such passwords, tokens, credentials or licences.
 - [ ] The blueprint uses `secret: true` in the `blueprint.yaml` parameter definition for question that ask for sensitive information.
 - [ ] The blueprint does not contain the files `xebialabs/.gitignore`, `xebialabs/values.xlvals` and `xebialabs/secrets.xlvals` and does not refer to them from the files section of the `blueprint.yaml` file. These files will be generted when the blueprint is instantiated. To generate the `xebialabs/values.xlvals` file, use the `saveInXlVals: true` directive in the parameters section of the blueprint. To generate the `xebialabs/secrets.xlvals` file, use the `secret: true` directive .
 - [ ] The blueprint does not define parameters for trivial things like phase names, task names, folder names etc. Ask questions that add value and be opinionated where possible. For example, ask for a project name and derive folder names, task names etc from that.
 - [ ] Folder and file names must use [kebab-case](http://wiki.c2.com/?KebabCase), for example: `aws/sample-app-demo`, `xld-environment.yaml`
- 
- 
+
+
 **Review and testing**
 
 This section describes how to determine whether the blueprint has been reviewed and tested well enough to be included in the central XebiaLabs blueprints repository.
 
 - [ ] The CI for the PR is green
 - [ ] The blueprint has been reviewed by someone else in the team.
+- [ ] The blueprint README has been reviewed by a Technical Writer and a Product Marketing Manager,.
```

Comment:
Do those titles include upper case letters? ;-)

Reply:
OK, maybe it's overkill. 😁